Ports/chocolate-doom/package.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='chocolate-doom'
+version='3.0.1'
+useconfigure='true'
+squashed_version=$(echo "${version}" | tr -d '.')
```

Comment:
This variable isn't used anywhere

Reply:
Oh yeah, that'd be an artifact of copying prboom's package.sh :)